frontend/app/views/spree/shared/_link_to_account.html.erb

```diff
@@ -5,3 +5,29 @@
   <%= link_to Spree.t('nav_bar.log_in'), spree_login_path, class: 'dropdown-item' if defined?(spree_login_path) %>
   <%= link_to Spree.t('nav_bar.sign_up'), spree_signup_path, class: 'dropdown-item' if defined?(spree_signup_path) %>
 <% end %>
+
+<% if try_spree_current_user %>
```

Comment:
instead of keepping logic in view, maybe you could implement here some helper ? I would suggest also some serializer for analytics data.